core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -714,76 +714,6 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
-  /api/admin/project/registration:
```

Comment:
These apis were removed because they required specific config I did not want to port and were only used by now defunct gpalloc. I scanned the logs of calls in broad-dsde-qa and broad-dsde-dev and found none.